shared/router-v2/tab-bar/tab-bar.css

```diff
@@ -81,6 +81,9 @@
   font-style: italic;
   letter-spacing: 0.1em;
 }
+.tab-selected .tab-encrypted {
+  color: #ffffff;
+}
```

Comment:
Do we also need styles for `.darkMode .tab-selected .tab-encrypted`?

Reply:
updated, better css